source/OptimaJet.DWKit.StarterApplication/Services/BuildEngine/BuildEngineSystemMonitor.cs

```diff
@@ -41,7 +41,12 @@ public void CheckBuildEngineStatus()
         }
         private async Task SyncDatabaseTablesAsync()
         {
-            var organizations = await OrganizationRepository.GetListAsync();
+            var organizations = await OrganizationRepository
+                .Get()
+                .Where(o => o.UseDefaultBuildEngine != null && o.UseDefaultBuildEngine.Value == false)
```

Comment:
Originally I had:
`.Where(o => o.useDefaultBuildEngine.GetDefaultValue() == false)`
But I noticed in the trace logs that this statement could be translated (I assume to SQL) and had to be run locally.

Reply:
that's a good "gotchya" for Entity Framework :+1: